CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 1.8.0-RC3
+
+- References librdkafka.redist 1.8.0-RC3. Refer to the [librdkafka release notes](https://github.com/edenhill/librdkafka/releases/tag/v1.8.0-RC3)
+for a complete list of changes, enhancements, fixes and upgrade considerations.
+- Added the `UseLatestVersion` configuration property to the Protobuf, JSON Schema and Avro serdes ([rayokota](https://github.com/rayokota)).
+
+## Fixes
+
+- **Breaking Change:** Updated the message framing format used by the Protobuf serdes to be compatible with the Java Protobuf serdes (message
```

Comment:
This needs more context, e.g., what type of clients (classes) that this applies to, and a proper mention of Schema-Registry.

Reply:
technically, this doesn't necessarily need to have anything to do with Confluent Schema registry. I'm not sure the change log is the place to write a tutorial on how all this works, but i'll add a note anyway.